modules/ROOT/pages/_partials/incpg-icr-conflict.adoc

```diff
@@ -181,7 +181,7 @@ remoteWins::
 
 ====
 =====
-// Inter-Sync Gateway Replication (v2) supports automatic conflict resolution to resolve conflicting document revisions.
+// Inter-Sync Gateway Replication supports automatic conflict resolution to resolve conflicting document revisions.
```

Comment:
```suggestion
// Inter-Sync Gateway Replication introduced in Sync Gateway 2.8 supports automatic conflict resolution to resolve conflicting document revisions.
```

GENERAL: If you prefer, you can also call out upfront at top of page that unless explicitly specified, all "inter-Sync Gateway replication" in page applies to Sync Gateway v2.8 and above.

Reply:
That sounds like a plan. I like it even better than my (pre 2.8 suggestion https://couchbase.slack.com/archives/CRWMUL4H5/p1602779797063200?thread_ts=1602766709.062900&cid=CRWMUL4H5)